### PR TITLE
Added Readme for helm

### LINKED ISSUE #353: add Readme to the current helm chart
The helm chart lacks documentation and how to install/customize it.

- [ ] Add [Readme](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/manifests/install/charts/as-a-second-scheduler/README.md) to the helm
- [ ] Update the [doc](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/doc/install.md#as-a-second-scheduler) specifying the steps to use helm chart -if any other place other than the above-

/good-first-issue
/kind-documentation

### PR DESCRIPTION
**Pull request by chetak123**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Fixes #353 

#### Special notes for your reviewer:
I have completed task 1 as specified in issue but not able to fully understand task 2 - `Update the` [doc](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/doc/install.md#as-a-second-scheduler) `specifying the steps to use helm chart -if any other place other than the above-`
It would be really helpful if you can please specify what needs to be done,
Thanks.

### CODE REVIEW

File: manifests/install/charts/as-a-second-scheduler/README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+| `controller.replicaCount`               | Controller replicaCount                                                                                                                   | `1`                                                     |
+| `plugins.enabled`                       | Plugins enabled                                                                                                                           | `["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch", "NodeResourcesAllocatable"]` |
+| `global.queueSort`                      | global queueSort, needs to be globally enabled once                                                                                       | `["Coscheduling"]`                                      |
+| `global.extensions.preFilter`           | global extensions preFilter                                                                                                               | `["Coscheduling", "CapacityScheduling"]`                |
```
Comment (helayoty) [inline manifests/install/charts/as-a-second-scheduler/README.md:38]: The description should be something like:
 ```markdown
 `preFilter` extension config. This is be used if the `preFilter` plugin is enabled.
 ```

Please apply the same for the rest of all extensions. parameters

File: manifests/install/charts/as-a-second-scheduler/README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+| `scheduler.name`                        | Scheduler name                                                                                                                            | `scheduler-plugins-scheduler`                           |
+| `scheduler.image`                       | Scheduler image                                                                                                                           | `k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.22.6`   |
+| `scheduler.namespace`                   | Scheduler namespace                                                                                                                       | `scheduler-plugins`                                     |
+| `scheduler.replicaCount`                | Scheduler replicaCount                                                                                                                    | `1`                                                     |
```
Comment (helayoty) [inline manifests/install/charts/as-a-second-scheduler/README.md:31]:
```suggestion
| `scheduler.replicaCount`                | scheduler-plugins replicas                                                                                                                | `1`                                                     |
```

File: manifests/install/charts/as-a-second-scheduler/README.md [hunk 3]
```diff
@@ -1 +1,45 @@
...
+| `controller.image`                      | Controller image                                                                                                                          | `k8s.gcr.io/scheduler-plugins/controller:v0.22.6`       |
+| `controller.namespace`                  | Controller namespace                                                                                                                      | `scheduler-plugins`                                     |    
+| `controller.replicaCount`               | Controller replicaCount                                                                                                                   | `1`                                                     |
+| `plugins.enabled`                       | Plugins enabled                                                                                                                           | `["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch", "NodeResourcesAllocatable"]` |
```
Comment (helayoty) [inline manifests/install/charts/as-a-second-scheduler/README.md:36]:
```suggestion
| `plugins.enabled`                       | All Plugins are enabled by default. Plugins enabled                                                                                                                           | `["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch", "NodeResourcesAllocatable"]` |
```

File: manifests/install/charts/as-a-second-scheduler/README.md [hunk 4]
```diff
@@ -1 +1,45 @@
...
+| `global.queueSort`                      | global queueSort, needs to be globally enabled once                                                                                       | `["Coscheduling"]`                                      |
```
Comment (helayoty) [inline manifests/install/charts/as-a-second-scheduler/README.md:37]: Please remove `global` from all parameters' descriptions. 
```suggestion
| `global.queueSort`                      |  THe default `queueSort` Plugin, needs to be enabled once                                                                                       | `["Coscheduling"]`                                      |
```

**Review by helayoty — CHANGES_REQUESTED** [review]
Thanks for your contribution. Added some comments.

**Comment by chetak123** [comment]
Hey @helayoty I have made the required changes.

(and I'm very sorry for the delay - due to exams)

File: manifests/install/charts/as-a-second-scheduler/README.md [hunk 5]
```diff
@@ -1 +1,45 @@
...
+
+
+#### Install chart using Helm v3.0+
+
+```bash
+$ git clone git@github.com:kubernetes-sigs/scheduler-plugins.git
+$ cd scheduler-plugins/manifests/install/charts
+$ helm install scheduler-plugins as-a-second-scheduler/
+```
+
```
Comment (seanmalloy) [inline manifests/install/charts/as-a-second-scheduler/README.md:21]:
````suggestion
Verify that scheduler and plugin-controller pod are running properly.

```bash
$ kubectl get deploy -n scheduler-plugins
NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
scheduler-plugins-controller   1/1     1            1           7s
scheduler-plugins-scheduler    1/1     1            1           7s
```
````

**Comment by seanmalloy** [comment]
@chetak123 I added one more suggestion to the README.

I believe we just need to change one thing. Currently the `helm install` commands are duplicated in the README you updated and in this document: https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/doc/install.md#as-a-second-scheduler.

Can you remove the `helm install` commands from `doc/install.md` and replace them with a link to the updated README?

**Review by helayoty — APPROVED** [review]
LGTM 🚢

**Comment by Huang-Wei** [comment]
/lgtm
/approve
/label tide/merge-method-squash

Thanks @chetak123 !

**Pull request merged**